admin/app/components/solidus_admin/ui/toast/component.rb

```diff
@@ -10,9 +10,12 @@ class SolidusAdmin::UI::Toast::Component < SolidusAdmin::BaseComponent
     ],
   }
 
+  ANIMATION_CLASSES = %w[translate-y-full opacity-0]
```

Comment:
Although we did it in other components I think it's better to switch to strings instead of arrays whenever we can so we don't put unnecessary pressure on memory and CPU